osu.Game/Screens/Edit/Menus/EditorMenuBar.cs

```diff
@@ -11,11 +11,19 @@
 using osu.Game.Graphics.UserInterface;
 using OpenTK;
 using OpenTK.Graphics;
+using System;
 
 namespace osu.Game.Screens.Edit.Menus
 {
     public class EditorMenuBar : OsuMenu
     {
+        /// <summary>
+        /// Invaoked when the selected mode has changed.

```

Comment:
Typo, should be `Invoked`

Reply:
Old commit - don't look at this for any editor changes, instead look at https://github.com/ppy/osu/pull/1337